modules/geo-layers/src/tile-layer/tile-2d-header.js

```diff
@@ -1,10 +1,11 @@
+import {log} from '@deck.gl/core';
+
 export default class Tile2DHeader {
```

Comment:
Here it is called header, not tile, so maybe a reason to align naming above?

Reply:
Cannot make a breaking change in 8.x.